modules/ztp-image-based-upgrade-with-backup.adoc

```diff
@@ -0,0 +1,229 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-with-backup_{context}"]
+= Upgrading the {sno} clusters with {lcao} with backup
+
+When you have generated the seed image and have completed the `Prep` stage, you can begin upgrading the target clusters.
+During the upgrade process, the OADP Operator creates a backup of the cluster, then {lcao} reimages it.
+
+[IMPORTANT]
+====
+If the upgrade fails or stops, you must roll back the upgrade.
+For more information, see "(Optional) Initiating rollback of the {sno} clusters after an image-based upgrade".
+
+In case of a rollback, all the resources created by the {lcao} are automatically deleted.
+====
+
+.Prerequisites
+
+* You have completed the `Prep` stage.
+* You have installed the OADP Operator.
+* You can an Object storage with your configured credentials.
+// Install OADP and storage configgged: Is it okay to have it in the prereqs? Or is this something that the user would expect to be guided through? With the different storage solutions, we probably can't cover everything.
+
```

Comment:
I do not think we should cover the installation of a S3 compatible object storage. We can link to ODF documentation for instance or MinIO since it is a certified OCP operator.